src/System.Device.Gpio/Interop/Unix/Libc/Interop.epoll_ctl.cs

```diff
@@ -2,7 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Files with Interop names are excluded from StyleCop by this line:
```

Comment:
nah, this is used for auto-generated files and it's side effect is disabling StyleCop can't you `#pragma disable SAXYZ` instead?

Reply:
I didn't know #pragma warning disable works for StyleCop rules. I only knew the SuppressMessage approach so far, but for these files that results in ugly code cluttering, so the autogenerated-approach was a workaround for this (see https://github.com/dotnet/iot/pull/892/commits/f0037bfdbfd243c5e36978e7d6a4244f8ae67193). 

Much better and cleaner now.